.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,49 @@
+name: Lint & Test
+
+on:
+  pull_request:
+    # edited is needed because that's the trigger when the base branch is
+    # changed on a PR
+    # The rest are the defaults.
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  lint_and_test:
+    name: Lint & Test
+    env:
+      CI: true
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16.x]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./.github/actions/setup
+      id: setup
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Run Flow
+      uses: Khan/flow-action@v1.1.3
+      with:
+        flow-bin: ./node_modules/.bin/flow
+      env:
+        ALL_CHANGED_FILES: '${{ steps.setup.outputs.added_modified }}'
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Run ESLint
+      uses: Khan/eslint-action@v2.0.0
+      with:
+        eslint-lib: ./node_modules/eslint
+      env:
+        ALL_CHANGED_FILES: '${{ steps.setup.outputs.added_modified }}'
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Run Jest
+      uses: Khan/jest-action@v1.2.0
```

Comment:
I've found these actions to not be as useful as just calling a script from our package.json. In fact in some repos, I've removed them and replaced with those scripts to decrease the number of runtime errors we got from them running (like permissions issues and such). What benefit do we get over just `yarn jest` at this point?

Reply:
So these actions give you inline annotation of errors! especially nice for flow & eslint, but also nice for jest imo

Reply:
I get that generally with the regular stuff - GitHub has updated significantly and we get the inline eslint stuff even when running it from yarn - and there's a way to customize its matchers to interpret the output for it to auto do this too without maintaining an action for it. Just wondering if we can drop maintenance of these actions in favor of the built-in stuff.

Reply:
oh cool! Yeah that would be lovely